src/foam/core/so/SystemNotificationServiceServer.js

```diff
@@ -91,6 +95,27 @@ foam.CLASS({
                 }
                 if ( ! match ) continue;
               }
+              if ( snt.getCidrWhiteList() != null &&
+                   snt.getCidrWhiteList().length > 0 ) {
+                boolean match = false;
+                foam.net.CIDR[] cidrs = snt.getCidrWhiteList();
+
+                String remoteIp = foam.net.IPSupport.instance().getRemoteIp(x);
+                ((foam.core.logger.Logger) x.get("logger")).warning(this.getClass().getSimpleName(), "getSystemNotifications", remoteIp);
```

Comment:
Why is this particular log a 'warning' ?

Reply:
It was for debugging, will remove